client/components/jetpack/profile-dropdown/index.tsx

```diff
@@ -39,7 +40,9 @@ const ProfileDropdown: React.FC = () => {
 
 	const trackedToggle = useTrackCallback( toggle, 'calypso_jetpack_masterbar_profile_toggle' );
 	const trackedClose = useTrackCallback( close, 'calypso_jetpack_masterbar_profile_close' );
-	const redirectToLogoutWithUser = useCallback( () => redirectToLogout( user ), [ user ] );
+	const redirectToLogoutWithUser = useCallback( () => dispatch( redirectToLogout() ), [
```

Comment:
The `*WithUser` suffix no longer makes sense. A better name would be `redirectToLogout`, but that conflicts with the action creator identifier...

Reply:
Renamed to `redirectToLogoutUrl` in https://github.com/Automattic/wp-calypso/pull/53661/commits/0f0fd40690beaca03598ab3f4b3c61653abc7d57 which I think makes more sense 😉